packages/core/src/Stores/client-store.js

```diff
@@ -567,7 +567,7 @@ export default class ClientStore extends BaseStore {
     @computed
     get is_valid_login() {
         if (!this.is_logged_in) return true;
-        const valid_login_ids_regex = new RegExp('^(MX|MF|VRTC|MLT|CR|FOG)[0-9]+$', 'i');
+        const valid_login_ids_regex = new RegExp('^(MX|MF|VRTC|VRDW|MLT|CR|FOG|AFF)[0-9]+$', 'i');
```

Comment:
for now I added this but there is a card in BE to fetch all valid accounts from Backend.